specs/validator/0_beacon-chain-validator.md

```diff
@@ -414,6 +421,10 @@ Where
 * `selection_proof` is the signature of the slot (`slot_signature()`).
 * `aggregate` is the `aggregate_attestation` constructed in the previous section.
 
+## Phase 0 attestation subnet stability
+
+Because Phase 0 does not have shards and thus does not have Shard Committees, there is no stable backbone to the attestation subnets (`committee_index{subnet_id}_beacon_attestation`). To provide this stability, each validator must randomly select and remain subscribed to `RANDOM_SUBNETS_PER_VALIDATOR` attestation subnets. The lifetime of each random subscription must be _at least_ `EPOCHS_PER_RANDOM_SUBNET_SUBSCRIPTION`.
```

Comment:
Can we add a note that this does not mean "`epoch % EPOCHS_PER_RANDOM_SUBNET_SUBSCRIPTION`"? Gradual changes (looking at the complete network) seem better here, we do not want everyone to search and switch on the exact same time.

Reply:
yeah, I was worried about the same thing. Mass exodus. I'll put that the lifetime should be between [EPOCHS_PER_RANDOM_SUBNET_SUBSCRIPTION, 2*EPOCHS_PER_RANDOM_SUBNET_SUBSCRIPTION]`

Reply:
updated